### ISSUE TITLE
Boxx layout is incorrect when text have ansi escape characters on it

### ISSUE THREAD

**Issue by syrusakbary** [EverlastingBugstopper/boxx#3]
Sometimes you want to put some color on the text inside of the Box.

Right now, if you do it `Boxx` will improperly calculate the length of the line where the colors are used.
See attached screenshot:

<img width="726" alt="Screen Shot 2020-02-18 at 10 39 11 AM" src="https://user-images.githubusercontent.com/188257/74767690-2c1f8500-523c-11ea-8bfc-f44130791c1a.png">

Some code that can trigger this is:

```rust
let message = format!(
        "There's a new version of the software! {} → {}\nUpdate with {}",
       "11".red(),
        "12".green(),
        "software self-update".green().bold()
    );
Boxx::builder().border_style(BorderStyle::Round).build().display(&message);
```

I think this can be easily solved by using a "cleaning" function over the provided text to `Boxx`, so ansi characters are cleaned out, similar to the following implementation in JS:

```js
/**
  This function removes the ansi escape characters
  (normally used for printing colors and so)
  Inspired by: https://github.com/chalk/ansi-regex/blob/master/index.js
  MIT License Copyright (c) Sindre Sorhus <sindresorhus@gmail.com> (sindresorhus.com)
 */
const cleanStdout = (stdout) => {
  const pattern = [
    "[\\u001B\\u009B][[\\]()#;?]*(?:(?:(?:[a-zA-Z\\d]*(?:;[-a-zA-Z\\d\\/#&.:=?%@~_]*)*)?\\u0007)",
    "(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PR-TZcf-ntqry=><~]))"
  ].join("|");

  const regexPattern = new RegExp(pattern, "g");
  return stdout.replace(regexPattern, "");
};
```

More context on the issue here:  https://github.com/wasmerio/wapm-cli/pull/169

**Comment by MarkMcCaskey** [comment]
Posted on the Wapm issue, but I'll post it here too, https://crates.io/crates/strip-ansi-escapes might be what you need here!

**Comment by EverlastingBugstopper** [comment]
Thanks for the reports and repros y'all - this should be fixed in `0.0.2-beta`, please let me know if you run into this again or any other issues 😄
